recipe/meta.yaml

```diff
@@ -8,11 +8,12 @@ package:
 source:
   fn: {{ name }}-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 4e913d807aa6598c320e8a50c50e2ee0602bc45240b485e3f8bc06f13060084c
+  sha256: 21f535de2e7a82c94c130c054adb5c7421d480d5619d61073996e2f89bcb879b
 
 build:
   number: 1
   script: {{ PYTHON }} -m pip install . --no-deps -vv
+  skip: true  # [py<38 or py>312]
 
 requirements:
   host:
```

Comment:
requires seutptools and setuptoools-scm in host
pywin32 >=226 in run

Reply:
went ahead and added wheel also